nx/lib/nx/binary_backend.ex

```diff
@@ -2,7 +2,7 @@ defmodule Nx.BinaryBackend do
   @moduledoc """
   An opaque backend written in pure Elixir that stores
   the data in Elixir's binaries.
-
+  
```

Comment:
You have a trailing whitespace here, that's what the formatter complains about :)